Tools/clinic/clinic.py

```diff
@@ -1713,7 +1714,11 @@ def render_option_group_parsing(
         count_min = sys.maxsize
         count_max = -1
 
-        add("switch (PyTuple_GET_SIZE(args)) {\n")
+        if limited_capi:
+            nargs = 'PyTuple_Size(args)'
+        else:
+            nargs = 'PyTuple_GET_SIZE(args)'
+        add(f"switch ({nargs}) {{\n")
```

Comment:
Style suggestion -- because of the f-strings needing the duplication of ``{``, perhaps just being explicit in both cases would be worth it?
```suggestion
            add("switch (PyTuple_Size(args)) {\n")
        else:
            add("switch (PyTuple_GET_SIZE(args)) {\n")
```